tests/read_port_mappings_test.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+
+try:
+
+    import sonic_platform_base.sonic_sfp.sfputilhelper
+except Exception as e:
+    print("Failed to load chassis due to {}".format(repr(e)))
+
+
+class TestPortMappingsRead(object):
+
+    self.platform_sfputil = None
+    self.test_dir = None
+    self.port_config = None
+
+    @classmethod
+    def setup_class(cls):
```

Comment:
Suggest replacing `setup_class` method with a Pytest fixture, as `setup_class` is more aligned with the `unittest` module, whereas fixtures are the Pytest-centric mechanism for setup/teardown.

Reply:
fixed